Added new reject and cancel feature for publish doc (#568)
appcircleio/appcircle-docusaurus: docs/publish-module/publish-information/reject-binary.md

---
title: Reject Binary
description: Learn how to reject binary in Appcircle
tags: [publish, publish details, publish flow, reject binary]
sidebar_position: 10
---

# Reject Binary

> Appcircle's **Reject Binary** feature allows you to reject the binary entering the publish process if it is not suitable. This feature adds flexibility to your publish management and allows you to manage your publish processes in more detail.

Reviewer: ```suggestion
Appcircle's **Reject Binary** feature allows you to reject the binary entering the publish process if it is not suitable. This feature adds flexibility to your publish management and enables more detailed control over your publishing processes.
```

> Appcircle's **Reject Binary** feature allows you to reject the binary entering the publish process if it is not suitable. This feature adds flexibility to your publish management and allows you to manage your publish processes in more detail.

Reviewer: ```suggestion
Appcircle's **Reject Binary** feature allows you to reject the binary entering the publish process if it is not suitable. This feature adds flexibility to your publish management and enables more detailed control over your publishing processes.
```

If there is a problem with the binary sent for publish or the metadata is invalid, you can use the **Reject Binary** feature to invalidate the binary by leaving a note so that your team does not make a mistake.

:::danger Reject Binary

When a binary file is **rejected**, it can no longer be [**Marked as Release Candidate**](/publish-module/publish-information/marking-release-candidates) and the rejection **cannot** be reversed.

:::

<Screenshot url='https://cdn.appcircle.io/docs/assets/BE4123-rejectAction.png' />

### Rejection Reason

> In order to use the Reject Binary feature, a rejection message must be left. This message will give other team members an idea of why the binary was rejected.

Reviewer: ```suggestion
To use the Reject Binary feature, you must provide a rejection message. This message will inform other team members of the reason for the rejection.
```
